src/components/FAQ.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { makeStyles } from '@material-ui/core/styles'
+import { GatsbyImage, getImage } from 'gatsby-plugin-image'
+import { formatImg } from 'src/utils/formatting'
+import Typography from '@material-ui/core/Typography'
+import faqPattern from 'src/img/causeshared/faqTitlePattern.png'
+import MuiAccordion from '@material-ui/core/Accordion'
+import MuiAccordionSummary from '@material-ui/core/AccordionSummary'
+import MuiAccordionDetails from '@material-ui/core/AccordionDetails'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+
+const useStyles = makeStyles((theme) => ({
+  parent: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '100%',
+    background: 'white',
+    alignItems: 'center',
+  },
+  row: {
+    width: '90%',
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'flex-start',
+    marginTop: theme.spacing(8),
+    marginBottom: theme.spacing(8),
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+    },
+  },
+  faqSection: {
+    display: 'flex',
+    flexDirection: 'column',
+    paddingLeft: theme.spacing(3),
+    [theme.breakpoints.down('sm')]: {
+      paddingLeft: theme.spacing(0),
+    },
+  },
+  titleImage: {
+    maxWidth: '35%',
+    [theme.breakpoints.down('sm')]: {
+      maxWidth: 'unset',
+    },
+  },
+  titleAndPattern: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  faqPattern: {
+    height: theme.spacing(12),
+  },
+  copy: {
+    marginTop: theme.spacing(3),
+    marginBottom: theme.spacing(3),
+  },
+  titleText: {
+    fontWeight: 'bold',
+    fontFamily: 'Poppins',
+  },
+  MuiAccordionRoot: {
+    '&.MuiAccordion-root:before': {
+      backgroundColor: 'white',
+      top: '0px',
+    },
+    '& .Mui-expanded': {
+      marginTop: '0px',
+    },
+    '& .MuiIconButton-label': {
+      color: theme.palette.secondary.main,
+    },
+    '& .MuiAccordionDetails-root': {
+      paddingTop: '0px',
+    },
```

Comment:
I don't see these classes being applied in the storybook, although they may be working locally before the production build step.  You'll want to apply it like this

```
MuiAccordionRoot: {
    '&::before': {
      backgroundColor: 'transparent',
      top: '0px',
    },
    
```
    In order to modify the icon, you'll want to to override the expandIcon classname in the Accordian Summary

Reply:
alternatively you can pass the color into the <ExpandMoreIcon/>